Jenkinsfile

```diff
@@ -1,6 +1,6 @@
 #!groovy
 
-library identifier: "jenkins-lib@develop"
+library identifier: "jenkins-lib@master"
```

Comment:
What's the purpose of this change?

Reply:
All of our other services use the master branch of jenkins-lib. The develop branch is behind master by a bit.